osquery/tables/utility/file_windows_ops.cpp

```diff
@@ -0,0 +1,111 @@
+
+#include "osquery\tables\utility\file_windows_ops.h"
+
+namespace osquery {
+
+/*
+        These functions were adapted from the Microsoft Windows Classic Samples
+   (MIT License)
```

Comment:
So I'm not great at code licensing, but I don't think we'll be able to bring in something that's explicitly licensed under MIT, as it's close to 2-clause BSD which we use, but there's extra things we don't license. I'd say we need to either re-write the code so it's our own to be licensed under our copyright, or we wont be able to use this :\

Reply:
Could this code possibly be re-written somehow so it's our own and we don't need to worry about the copyright?

Reply:
So the code was adapted from here https://github.com/Microsoft/Windows-classic-samples/blob/master/Samples/Win7Samples/winbase/io/extendedfileapis/ExtendedFileAPIs.cpp but I didn't modify the usage to return an attribute string. Does that work? It's really just the condional structure that I used. If not I'll see what I can do to get to the same result.